parallel_tests.gemspec

```diff
@@ -10,5 +10,6 @@ Gem::Specification.new name, ParallelTests::VERSION do |s|
   s.license = "MIT"
   s.executables = ["parallel_spinach", "parallel_cucumber", "parallel_rspec", "parallel_test"]
   s.add_runtime_dependency "parallel"
+  s.add_runtime_dependency "sys-proctable"
```

Comment:
If you're using bundler test groups, it is a dependency that only applies to your test environment.  The point about it being mac/unix only is fair.  Do people run Ruby test in Windows environments?